leveldbstatic/include/port/port_config.h

```diff
@@ -7,7 +7,7 @@
 
 // Define to 1 if you have a definition for fdatasync() in <unistd.h>.
 #if !defined(HAVE_FDATASYNC)
-#define HAVE_FDATASYNC 1
+#define HAVE_FDATASYNC 0
```

Comment:
Why are we manually setting this?

Reply:
This file no longer needs to be checked in, if we're always calling cmake, it will just generate this file for you.